src/scripting/game_lua_kernel.cpp

```diff
@@ -3089,7 +3089,13 @@ int game_lua_kernel::intf_has_achievement(lua_State *L)
 {
 	const char *content_for = luaL_checkstring(L, 1);
 	const char *id = luaL_checkstring(L, 2);
-	lua_pushboolean(L, preferences::achievement(content_for, id));
+
+	if(resources::controller->is_networked_mp() && synced_context::is_synced()) {
+		ERR_LUA << "Returning true for whether a player has completed an achievement due to being networked multiplayer.";
+		lua_pushboolean(L, true);
```

Comment:
Wouldn't false be a more natural fallback here?

Reply:
Updated.